src/utils/static-cache.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
We might want to update this file on a monthly or weekly basis using GitHub Action, but let's validate this idea first.